ephios/api/urls.py

```diff
@@ -15,11 +15,23 @@
     ApplicationUpdate,
 )
 from ephios.api.views.events import EventViewSet
-from ephios.api.views.users import UserProfileMeView
+from ephios.api.views.users import (
+    MailBasedUserParticipationView,
+    UserParticipationView,
+    UserProfileMeView,
+)
 from ephios.extra.permissions import staff_required
 
 router = routers.DefaultRouter()
 router.register(r"events", EventViewSet)
+router.register(
+    r"users/(?P<user>[\d]+)/participations", UserParticipationView, basename="user-participations"
```

Comment:
Why doesn't this route show up in the endpoint directory at http://localhost:8000/api/?

Reply:
I don't know where that is coming from, it shows up in http://localhost:8000/api/schema/

Reply:
could you research :finger_right: :finger_left:

Reply:
because DRF displays hyperlinks to all list views on the router. As there is an url param in the prefix, DRF has no chance of rendering one working hyperlink to this view